db/seeds.rb

```diff
@@ -1,362 +1,528 @@
-          [
-            ['Describe your eggyolk\'s work-life balance.', 'work_life_balance', rockclimbing_skill, :free],
-            ['Is your eggyolk often exhausted?', 'is_exhausted', rockclimbing_skill, :dropdown, ['Yes', 'No'], no_dependents],
-            nil
-          ]
-        ]
-      ]
-    ],
-    [
-      'Does your eggyolk have any allergies?', 'has_allergies', eggyolk_information, :dropdown,
-      ['Yes', 'No'],
-      [['List any foods your eggyolk may be allergic to.', 'allergic_to', eggyolk_information, :free], nil] 
-    ],
-    ['What is your eggyolk\'s budget on rockclimbing?', 'rockclimbing_budget', eggyolk_information, :range,
-      [[0, 100], [101, 500], [501, safe_maximum]],
-      [nil, nil, ['Is your eggyolk employed?', 'is_employed', eggyolk_information, :dropdown, ['Yes', 'No'], no_dependents]]
-    ],
-    ['Briefly describe how your eggyolk discovered rockclimbing.', 'rockclimbing_discovery', eggyolk_information, :free],
-    ['Explain why your eggyolk enjoys rockclimbing, if applicable.', 'reasons_to_rockclimb', eggyolk_information, :free],
-    ['What kind of shoes does your eggyolk use when rockclimbing?', 'rockclimbing_shoes', eggyolk_information, :dropdown,
-      ['Laces', 'Velcro', 'Slipper'], no_dependents
-    ],
-    ['Which kind of rockclimbing does your eggyolk engage in?', 'rockclimbing_types', eggyolk_information, :dropdown,
-      ['Bouldering', 'Top Rope Climbing', 'Mountaineering'], no_dependents
-    ],
-    ['How many shoes does your eggyolk own?', 'num_shoes', eggyolk_information, :range, [[0, 10]], no_dependents],
-    ['What size shoe is your eggyolk?', 'shoe_size', eggyolk_information, :range, [[0, 20]], no_dependents]
-  ]
-end
-
-# Questions
-# -----
-# FORMAT
-# [<text>, <parameter>, <category>, <type>, <array of options>, <array of dependents (or nils)>]
-
-def generate_question(question, building_type, parent_option=nil)
-  if question.nil?
-    return
-  end
-
-  q_text = question[0]
-  q_parameter = question[1]
-  q_category = question[2]
-  q_type = question[3]
-  q_options = question[4]
-  q_dependents = question[5]
-
-  q = Question.create!(
-    question_type: q_type,
-    status: :published,
-    text: q_text,
-    building_type: building_type,
-    category: q_category,
-    parameter: q_parameter
-  )
-
-  unless parent_option.nil?
-    q.parent_option = parent_option
-    q.save!
-  end
-
-  if q_type.equal? :free
-    return
-  end
-
-  if q_type.equal? :dropdown
-    q_options_saved = q_options.map do |option|
-      DropdownOption.create!(
-        text: option,
-        question: q
-      )
-    end
-  end
-  if q_type.equal? :range
-    q_options_saved = q_options.map do |option|
-      RangeOption.create!(
-        min: option[0],
-        max: option[1],
-        question: q
-      )
-    end
-  end
-
-  Array(0...q_options_saved.length).each do |i|
-    generate_question(q_dependents[i], building_type, q_options_saved[i])
+# # Instantiate Faker's RNG
```

Comment:
Let's remove the old seed code that is currently commented out. Unless there is a continuing need for it?